game/script-topics.rpy

```diff
@@ -1537,6 +1537,80 @@ label talk_driving:
             else:
                 n "That's what friends are for, [player]!"
 
+# Natsuki laments her inability to drive and questions the player on if they can
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._topic_database,
+            label="talk_gaming",
+            unlocked=True,
+            prompt="Are you into video games?",
+            conditional=None,
```

Comment:
Don't think we need to specify `conditional=None` as this should be the default value already if I'm correct.